modoboa/core/views/dashboard.py

```diff
@@ -43,11 +45,13 @@ def get_context_data(self, **kwargs):
                 self.request.localconfig.parameters.get_value("rss_feed_url"))
             if custom_feed_url:
                 feed_url = custom_feed_url
-        posts = feedparser.parse(feed_url)
+        if not settings.DISABLE_RSS:
+            posts = feedparser.parse(feed_url)
         entries = []
-        for entry in posts["entries"][:5]:
-            entry["published"] = parser.parse(entry["published"])
-            entries.append(entry)
+        if not settings.DISABLE_RSS:
+            for entry in posts["entries"][:5]:
+                entry["published"] = parser.parse(entry["published"])
+                entries.append(entry)
```

Comment:
To avoid 2 checks, you could rewrite your code as follows:
```python
entries = []
if not settings.DISABLE_RSS:
    posts = feedparser.parse(feed_url)
    for entry in posts["entries"][:5]:
        entry["published"] = parser.parse(entry["published"])
        entries.append(entry)